### ISSUE TITLE
Place dirt on grass in area mode

### ISSUE THREAD

**Issue by JulianGro** [nicguzzo/wands#31]
Currently when trying to terraform using the area mode, you cannot just lift the ground because the wand will specifically ask you for grass, when it should really just use dirt.

**Comment by nicguzzo** [comment]
Yes, that's intended behavior, I'm also working on a way to use the palette as a kind of selection for placement, e.g. if you want to place stone over a wall made of diorite and andesite, you would place andesite and diorite on a palette in the offhand and stone on a wand slot (as a reference block, the same way as the palette, not for storage), and even a second palette to serve as the source
